### PR TITLE
[8.0] Standard naming for TokenManager service classes

### PR DESCRIPTION
**Pull request by atsareg**
This PR is a follow-up of #7888 to make names for the service handlers following the standard convention. The PR propagated to integration replaces #7893. This also means that upon the update the DIRAC installations using already the Tornado TokenManager service will have to update the TokenManager service configuration by adding the HandlerPath option:

```
HandlerPath = DIRAC/FrameworkSystem/Service/TornadoTokenManagerHandler.py
``` 

This will be reflected in the release notes and instructions

BEGINRELEASENOTES

*Framework
CHANGE: Rename TokenManager service classes to follow standard convention for tornado and diset handlers

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/FrameworkSystem/ConfigTemplate.cfg [hunk 1]
```diff
@@ -28,6 +28,20 @@ Services
   ##BEGIN TokenManager:
   # Section to describe TokenManager system
   TokenManager
+  {
+    Port = 9180
```
Comment (fstagni) [inline src/DIRAC/FrameworkSystem/ConfigTemplate.cfg]: 9180 is taken, maybe 9181?

File: src/DIRAC/FrameworkSystem/Service/TornadoTokenManagerHandler.py [hunk 2]
```diff
@@ -1,6 +1,8 @@
-"""TokenManager service is responsible for token management, namely storing, updating,
+"""TokenManager service is a HTTPs-exposed service responsible for token management, namely storing, updating,
```
Comment (fstagni) [inline src/DIRAC/FrameworkSystem/Service/TornadoTokenManagerHandler.py]:
```suggestion
"""TornadoTokenManager service is a HTTPs-exposed service responsible for token management, namely storing, updating,
```

File: src/DIRAC/FrameworkSystem/Service/TornadoTokenManagerHandler.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
 requesting new tokens for DIRAC components that have the appropriate permissions.
 
+.. note:: As a newly created service, it will not support the old DIPS protocol, which is living to its age.
```
Comment (fstagni) [inline src/DIRAC/FrameworkSystem/Service/TornadoTokenManagerHandler.py]: I would remove this...

File: src/DIRAC/FrameworkSystem/ConfigTemplate.cfg (hunk 1 above)
Comment (atsareg) [reply src/DIRAC/FrameworkSystem/ConfigTemplate.cfg]: OK

**Review by fstagni — APPROVED** [review]

**Pull request merged**